lib/Runtime/Base/JnDirectFields.h

```diff
@@ -279,6 +279,8 @@ ENTRY(now)
 ENTRY(null)
 ENTRY(Number)
 ENTRY(number)
+ENTRY(BigInt)
+ENTRY(bigint)
```

Comment:
what is the lowercase `bigint` used for?

Reply:
will remove